index.rst

```diff
@@ -49,9 +49,11 @@ Safety: Quick Reference for Nighttime Personnel
 It does not replace the required safety training to be on the summit at night. 
 This includes:
 
-- Procedures on who (and how) to call when there is an issue
-- What to do in case of vehicle failure, power-outage, fire etc.
-- Emergency meeting locations
+- procedures on who (and how) to call when there is an issue
```

Comment:
something weird happened with these bullets.
this is repeated by line 55.

Also, the first letters of a bullet should be capitalized.

Reply:
I don't believe I edited these.

Reply:
In this branch, this looks different, but I have modified with your suggestion.